integr8ly/integreatly-operator
# INTLY-2757 user level sso

### Description 
Add launcher operator 
Manifest PR: https://github.com/integr8ly/manifests/pull/59

### Verification steps
- Create the operator source which points to my app registry:
```yml
apiVersion: operators.coreos.com/v1
kind: OperatorSource
metadata:
  finalizers:
    - finalizer.operatorsources.operators.coreos.com
  name: integreatly-operators
  namespace: openshift-marketplace
spec:
  authorizationToken: {}
  displayName: Integreatly Operators austincunningham
  endpoint: 'https://quay.io/cnr'
  publisher: austincunningham Publisher
  registryNamespace: austincunningham
  type: appregistry
```
- Create a subscription to the Integreatly operator from the Operator Hub, then create an `installation` CR 
```yml
apiVersion: integreatly.org/v1alpha1
kind: Installation
metadata:
  name: example-installation
spec:
  type: workshop
  namespacePrefix: integreatly-
  routingSubdomain: apps.cluster-id.openshiftworkshop.com
  masterUrl: https://console-openshift-console.apps.cluster-id.openshiftworkshop.com
  apiUrl: https://api.cluster-id.openshiftworkshop.com
  selfSignedCerts: true
```

- Verify that the `integreatly-user-sso` namespace is created
- Verify that the `rhsso-operator` pod in the namespace 
- Verify that the user-sso route is accessible 
- Verify that the you can login to the realm user-sso using the sso admin credentials
